linkerd/app/inbound/src/http/router.rs

```diff
@@ -131,19 +116,9 @@ impl<C> Inbound<C> {
                         .http_endpoint
                         .to_layer::<classify::Response, _, _>(),
                 )
-                .push_on_service(
-                    svc::layers()
-                        .push(http_tracing::client(
-                            rt.span_sink.clone(),
-                            super::trace_labels(),
-                        ))
-                        .push(http::BoxResponse::layer())
-                        // This box is needed to reduce compile times on recent
-                        // (2021-10-17) nightlies, though this may be fixed by
-                        // https://github.com/rust-lang/rust/pull/89831. It should
-                        // be removed when possible.
-                        .push(svc::BoxService::layer())
```

Comment:
i'm assuming this `BoxService` was obviated by the boxing added in this PR?

Reply:
We're just doing this everywhere now, so this place isn't all that special. The boxing is now included in the arc_ helper